optimizer/lib/transformers/BlurImagePlaceholders.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+var sizeOf = require('image-size');
```

Comment:
s/var/const - Also, can you ensure that those dependencies are in package.json?

Reply:
I'll definitely ensure that they get added! And sorry for the ignorance, but does "s/var/const" mean to use "const sizeOf" instead of "var sizeOf"? Sorry just want to ensure I don't make the wrong change haha.

Reply:
Sorry I wasn't clear. Yes, it means replace "var" with "const"